src/components/application_manager/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -78,7 +81,62 @@ class OnSystemRequestNotificationTest
   MockMessageHelper& message_helper_;
 };
 
-TEST_F(OnSystemRequestNotificationTest, Run_ProprietaryType_SUCCESS) {
+#ifdef EXTENDED_POLICY
+bool CheckHeader(MessageSharedPtr msg) {
+  struct DataValidator {
+    bool& is_valid_;
+    const utils::json::JsonValue& json_data_;
+
+    DataValidator(bool& is_valid, const utils::json::JsonValue& data)
+        : is_valid_(is_valid = true), json_data_(data) {}
```

Comment:
@LevchenkoS The syntax `is_valid_(is_valid = true)` is confusing. Please consider separate initialization in the constructor.